setup.cfg

```diff
@@ -2,7 +2,7 @@
 max-line-length = 90
 max-complexity = 10
 exclude = *migrations*
-ignore = D100,D203,D204
+ignore = D100,D104,D203,D204
 statistics=true
 application-import-names = pgcrypto
-import-order-style = google
+import-order-style = smarkets
```

Comment:
It has been added in https://github.com/public/flake8-import-order/pull/57

> smarkets - style as google only with import statements before from X import

I'm happy they've made the move to keep compability